src/printer.js

```diff
@@ -1,4 +1,46 @@
 // https://prettier.io/docs/en/plugins.html#printers
-const genericPrint = (path, options) => options.originalText;
+const { concat, line } = require('prettier').doc.builders;
+
+const space = ' ';
+
+const genericPrint = (path, options) => {
+  const node = path.getValue();
+  const lines = options.originalText.split('\n');
+  const parts = [];
+
+  switch (node.type) {
+    case 'SourceUnit':
+      node.children.forEach(child => {
+        switch (child.type) {
+          case 'PragmaDirective':
+            parts.push(
+              concat([
+                'pragma',
+                space,
+                child.name,
+                space,
+                child.value,
+                ';',
+                line,
+                line
+              ])
+            );
+            break;
+          default:
```

Comment:
I will add more and more cases and recursively call print to handle them all while default should not change too much I hope 😅